dotcom-rendering/scripts/check-node-versions.mjs

```diff
@@ -3,13 +3,13 @@
 import { readFile } from 'node:fs/promises';
 import { dirname, resolve } from 'node:path';
 import { fileURLToPath } from 'node:url';
-import { log, warn } from '../dotcom-rendering/scripts/env/log.js';
+import { log, warn } from '../../scripts/log.js';
 
 const __dirname = dirname(fileURLToPath(import.meta.url));
 
 process.chdir(resolve(__dirname, '..'));
 
-const nvmrc = (await readFile('.nvmrc', 'utf-8'))
+const nvmrc = (await readFile('../.nvmrc', 'utf-8'))
```

Comment:
I’m not sure this should live inside `dotcom-rendering` as it’s looking for files in both workspaces and in the root.

Reply:
it's a DCR concern though right? it's looking in the root for the root version, and making sure that DCR-related files (which AR files are, really, i think?) are in sync with the repo more generally. 

should it be the responsibility of the root to ensure that apps in the workspace stay in sync with the root?

Reply:
In my mind the root is a good place for scripts that span all workspaces, but I agree that it should be a workspace concern to stay in sync with the root.

I’ll defer to anyone in @guardian/dotcom-platform with a stronger or better articulated view.

Reply:
Merged, but if anyone prefers to move it, please be my guest